ReactWindows/ReactNative.Shared/Modules/Network/NetworkingModule.cs

```diff
@@ -485,6 +485,32 @@ private static void ApplyHeaders(HttpRequestMessage request, string[][] headers)
                 var key = header[0];
                 switch (key.ToLowerInvariant())
                 {
+                     case "authorization":
+                        //create scheme from user's Authorization first word
+                        string createScheme = header[1].Split(' ')[0];
+
+                        //Remove above added scheme from authorization because it's already added
+                        string completeToken = header[1].Replace(header[1].Split(' ')[0], "");
```

Comment:
You're calling the `.Split()` and `.Trim()`, rather than using the value that gets created above (i.e., `createScheme`)

Reply:
Thanks @rozele for your review comments.In the beginning i tried that also but  HttpCredentialsHeaderValue method accepts only string without spaces.